### PR TITLE
add pajlang

### PR DESCRIPTION
**Pull request by gempir**
Add support for one of the greatest programming lanugages of all time. 

**pajlang**

 https://github.com/gempir/pajlang

### CODE REVIEW

File: adapters/paj.paj [hunk 1]
```diff
@@ -0,0 +1,2 @@
+$var = "Connection established";
+print($var);
```
Comment (RAnders00) [inline adapters/paj.paj:2]: There is no newline at the end of this file. ian-db follows rigurous quality standards. Please correct.

**Review by RAnders00 — CHANGES_REQUESTED** [review]

File: adapters/paj.paj (hunk 1 above)
Comment (gempir) [reply adapters/paj.paj:2]: image

Comment (RAnders00) [reply adapters/paj.paj]: rocket emoji

**Review by RAnders00 — APPROVED** [review]

**Comment by CBenni** [comment]
I do sense some bones in this, can you please make sure to remove them and create some tests? 
Else this will break my XML-over-MMS interface. Thanks

**Comment by gempir** [comment]
This has been in review by an external security contractor with military grade security research. I assure you no tests are required. And it will only enhance your XML-over-MMS interface

**Comment by RAnders00** [comment]
I also have to vouch for the pajlang integration. At our company developer productivity has increased 420% and we have doubled our MMS/second rates.

**Comment by CBenni** [comment]
Interesting - @RAnders00  are you also sending the XML as gif files? Or do you prefer .mp3?

**Review by styler — APPROVED** [review]
Wow, really nice PR!!!

**Comment by RAnders00** [comment]
mp3 is sooooo 2014 @CBenni what are you, old? LOL

**Pull request merged**